stl_text/ops/tokenizers/whitespace_tokenizer.py

```diff
@@ -1,44 +1,52 @@
+import torch
 import torch.nn as nn
 from typing import Dict, List, Optional
+from collections import OrderedDict
 from iopath.common.file_io import PathManager
+from torchtext.experimental.vocab import (
+    build_vocab_from_text_file,
+    Vocab,
+)
+from torchtext._torchtext import Vocab as VocabPybind 
 
+class SplitTextModule(nn.Module):
+    def __init__(self):
+        super(SplitTextModule,self).__init__()
 
```

Comment:
I am trying to see if explicitly creating Module for split() can be avoided. The reason I have to do this is build_vocab_from_text_file require jitable tokenizer.

Reply:
You can try [load_vocab_from_file](https://github.com/pytorch/text/blob/master/torchtext/experimental/vocab.py#L53) which simply loads the vocab column from a txt file.